README.md

```diff
@@ -1 +1,53 @@
 Assignment submissions here
+
+Q.0-1] ls , cat readme
+Q.1-2] ls , cat ./-
+Q.2-3] cat ‘spaces in the file name’ aBZ0W5EmUfAf7kHTQeOwd8bauFJ2lAiG
+Q.3-4]ls, ls -a, cd inhere, cat .hidden 2EW7BBsr6aMMoJ2HjW067dm8EgX26xNe
+Q. 4-5]file  cd inhere , file ./* , cat ./-file07                        lrIWWI6bB37kxfiCQZqUdOIYfr6eEeqR
+Q.5-6] ls , cd inhere, ls , find . -size 1033c  , cat ./maybehere07/.file2.
+   Password : P4L4vucdmLnm8I7Vl7jG1ApGSfjYKqJU
+
+Q.6-7] ls , cd inhere , file./* , cat ./-file07                					z7WtoNQU2XfjmMtWA8u5rN4vzqu4v99S      
+Q.7-8] 	data.txt grep millionth			        TESKZC0XvTetK0S9xNwm25STk5iWrBvP        
+Q.8-9] sort data.txt , uniq data.txt                        EN632PlfYiZbn3PhVK3XOGSlNInNE00t
+Q.9-10] strings  grep = data.txt			G7w8LIi6J3kTb8A7j9LgrywtEUlyyp6s
+Q.10-11] base64 - -decode data.txt			6zPeziLdR2RKNdNYFNb6nVCKzphlXHBM
+Q.11-12]  cat data.txt | tr a-zA-Z n-za-mN-ZA-M	JVNBBFSmZwKKOP0XbFXOoW8chDz5yVRv		
+Q.12-13]ls, mkdir /tmp/mdg , cd /tmp/mdg, cp ~/data.txt, ls, mv data.txt mdg1, xxd -r mdg1 > mdg2 , file mdg2 , mv mdg2 .gz , gunzip mdg2.gz ,  repeat these zipping until you get the file.                                                                     wbWdlBxEir4CaE8LaPhauuOo6pwRmrDw
+
+
+Q 13-14].Do same as done in lvl 16-17 after we gett the rsa private key 			fGrHPx402xGC7U7rXKDaxiWFTOiF0EN
```

Comment:
How do you get the private key here? Also write the exact commands used in this level